src/test/java/flashcard/FlashcardDeckTest.java

```diff
@@ -0,0 +1,44 @@
+package flashcard;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class FlashcardDeckTest {
+
+    @Test
+    void addCards() {
```

Comment:
You should rename your test function name in the following format!
methodUnderTest_inputGiven_ExpectedOutput

Reply:
Just adding on, the more general alternative would be testAddCard() !

Remember to apply the renaming to delete card and write to file as well.